jwst/cube_build/ifu_cube.py

```diff
@@ -714,7 +714,7 @@ def setup_IFUCube(self,j):
         IFUCube.meta.wcsinfo.ctype3 = 'WAVE'
         IFUCube.meta.wcsinfo.cunit3 = 'um'
         IFUCube.meta.wcsinfo.wcsaxes = 3
-        IFUCube.meta.wcsinfo.pc1_1 = 1
+        IFUCube.meta.wcsinfo.pc1_1 = -1
```

Comment:
The values of exactly 0 and +-1 for the PC matrix terms suggest that the data are oriented with North up. Are the cube data in fact always rotated so that North is up? If not, the PC matrix terms need to reflect the orientation on the sky.

Reply:
On a few IFUcubes (in the Northern and Southern hemisphere)  I did the simple check about looking at the same column of data  in the IFUCube and verifying that ra did not change and then the same row in the IFUcube to verify the dec did not change.  In the current implementation North is always up. I will add to an issue that in the future we probably want to do what resample for images does and have an option to have the IFUCube rotated on the sky from North more in alignment with the actual footprint.